src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -397,6 +397,16 @@ public bool ShouldVersionOldDocument(DocumentsOperationContext context, Document
             Debug.Assert(changeVector != null, "Change vector must be set");
             Debug.Assert(lastModifiedTicks != DateTime.MinValue.Ticks, "last modified ticks must be set");
 
+            if (flags.Contain(DocumentFlags.FromReplication) == false &&
```

Comment:
This doesn't match what is written in the issue?

Reply:
not sure I follow :-)

Reply:
```
- Add the id check for putting the revision (512 bytes)
- Increase the check on read for revision tombstone to 1024 bytes
```

Reply:
> ```
> - Add the id check for putting the revision (512 bytes)
> ```
done ? :)

> ```
> - Increase the check on read for revision tombstone to 1024 bytes
> ```
this is only relevant to v6.0 - we are only using `Slice.From` for revision tombstones here

Reply:
done